packages/ibm-products/src/components/ExportModal/ExportModal.tsx

```diff
@@ -49,7 +49,8 @@ type PreformattedExtensions = {
   extension?: string;
   description?: string;
 };
-interface RemoveModalProps extends React.ComponentProps<typeof ComposedModal> {
+export interface ExportModalProps
```

Comment:
I also renamed this to ExportModalProps it was ~~RemoveModalProps ~~. Don't know if it was a typo before.